src/test/java/core/basesyntax/strategy/operation/BalanceOperationHandlerTest.java

```diff
@@ -0,0 +1,72 @@
+package core.basesyntax.strategy.operation;
+
+import static org.junit.Assert.assertEquals;
+
+import core.basesyntax.Utils;
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.FruitTransaction;
+import org.junit.After;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class BalanceOperationHandlerTest {
+    private static final String FRUIT = "banana";
+    private static final String EMPTY = "";
+    private static final int EXPECTED = 20;
+    private static FruitTransaction transaction;
+    private static BalanceOperationHandler balanceOperation;
+
+    @BeforeClass
+    public static void beforeAll() throws Exception {
+        Storage.fruits.clear();
```

Comment:
you should clear storage after each test in services that deal with storage

not after all tests or before all tests

so you should be able to remove this line from here